samples/graph-notification-team-channel/csharp/GraphNotification/Manifest/manifest.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
```

Comment:
in .gif image, we have missed the step to delete/update/ the team or channels.
![image](https://user-images.githubusercontent.com/59052547/186747552-6806c20e-1530-4169-8f19-b570390eb587.png)

Reply:
I will update with that two images.